mtggoldfish.py

```diff
@@ -148,16 +168,21 @@ def save_deck_to_cache(deck, deck_id):
     todays_date = datetime.now()
     month = todays_date.month
     day = todays_date.day
-    if month <= 9: month = "0%s" %(todays_date.month)
-    if day <= 9: day = "0%s" %(todays_date.day)
-    cache_file_name = "%s_%s_%s_%s" %(deck_id, month, day, todays_date.year)
+    if month <= 9:
+        month = "0%s" % (todays_date.month)
+    if day <= 9:
+        day = "0%s" % (todays_date.day)
+    cache_file_name = "%s_%s_%s_%s" % (deck_id, month, day, todays_date.year)
 
     with open(os.path.join(cache_dir, cache_file_name), 'wb') as output:
         pickle.dump(deck, output, pickle.HIGHEST_PROTOCOL)
 
+
 """
 Given a DeckID, load the deck from the cache
 """
+
```

Comment:
Are these newlines before a function name really part of PEP-8? Some things in PEP-8 I disagree with :/. I mean if it's the standard then I'm not going to deny the pull request for it, but I figured I'd voice my opinion on it

Reply:
Yes, two blank lines before a top level class or definition, one line after. It's supposed to improve readability, although I take it mostly because my editor will tell me so :)